composer.json

```diff
@@ -19,7 +19,7 @@
         "symfony/property-access": "^3.2|^4",
         "pagerfanta/pagerfanta": "^1.0.5|^2.0",
         "psr/log": "^1.0",
-        "ruflin/elastica": "^5.2.1|^6.0"
+        "ruflin/elastica": "dev-master"
```

Comment:
We would like to keep stable packages.

Reply:
Completely agreed, alas last release from ruflin/elastica is from waaay back
> Release 6.0.2 @ruflin ruflin released this on May 29 · 8 commits to master since this release

See link for reference https://github.com/ruflin/Elastica/releases

And the commit that's needed for this PR, is a fairly recent one on master 

https://github.com/ruflin/Elastica/commit/d6f14758126fbec14fd5d8b6bf0001a7bbd97591

I'm not sure what's the best way around this? ask ruflin to release a new version?

Reply:
@ruflin nothing would make me more happy – but i'm not sure what your release cycle is like/don't want to bother if you're waiting for something else to merge in

Reply:
@ruflin yeah let's have a new ruflin/Elastica release, if that's okay – it'll push this PR forward

Reply:
What is the urgency on this one? Probably only get to do a release beginning next week.

Reply:
Next week is totally appropriate

Reply:
@ruflin super-gentle nudge if there are any updates on the new release of ruflin/Elastica?

Reply:
Thanks for the nudge ;-) Just opened the PR to prepare the release. Now I only need a review for it: https://github.com/ruflin/Elastica/pull/1547

Reply:
@jandom This should help: https://github.com/ruflin/Elastica/releases/tag/6.1.0

Reply:
Yaaas thank you!